src/desktop/apps/article/routes.ts

```diff
@@ -212,6 +214,42 @@ export const classic = (req, res, _next) => {
   })
 }
 
+function prepForAMP(article) {
```

Comment:
This should live with the article helpers here: https://github.com/artsy/force/blob/master/src/desktop/apps/article/helpers.tsx

Reply:
Is this not shared between server and client? (I guess I can just look that up.)

Reply:
Alas these helpers are also imported in some client-side JS files, which would again pull `cheerio` into those bundles. I was unsure about creating a new file, so instead I’m leaving it here and adding a comment about why it does not exist in the shared helpers file.

Reply:
@eessex This piece of info is key ☝️